src/modules/assignment/assignment.controller.ts

```diff
@@ -12,23 +12,27 @@ import {
 } from '@nestjs/common';
 import { AssignmentService } from './assignment.service'; //eslint-disable-line 
 import { CreateAssignmentDTO } from './dto/create-assignment.dto'; //eslint-disable-line 
-import { ApiCreatedResponse, ApiProperty } from '@nestjs/swagger';
+import { ApiCreatedResponse, ApiProperty, ApiTags } from '@nestjs/swagger';
 import { UpdateAssignmentDTO } from './dto/update-assignment.dto';
+import { IsEmail, Length } from 'class-validator';
 
 class AssignmentResponseBody {
   @ApiProperty({ required: true, example: '605e3fd9acc33583fb389aec' })
   id: string;
 
   @ApiProperty({ required: true, example: 'Noob' })
+  @Length(3, 20)
```

Comment:
The class validators will have no effect here, Please remove the class validators from the swagger response body.